src/dll.py

```diff
@@ -0,0 +1,78 @@
+class Node(object):
+    """Node class."""
+
+    def __init__(self, data, prev=None, next=None):
+        """Initialize Node."""
+        self.data = data
+        self.prev = prev
+        self.next = next
+
+
+class DoublyLinkedList(object):
+    """Doubly linked list class."""
+
+    def __init__(self, iterable=None):
+        """Intialize DLL."""
+        self.head = None
+        self.tail = None
+        self._size = 0
+        if iterable is not None:
+            for item in iterable:
+                self.push(item)
+
+    def push(self, data):
+        """Push data into doubly linked list."""
+        node = Node(data)
+        if self.head is None:
+            self.head = node
+            self.tail = node
+        else:
+            self.head.prev = node
+            node.next = self.head
+            self.head = node
```

Comment:
Since this line is repeated on both sides of your if/else ternary operator, you can just type it in once after the if/else block.